src/components/radio-group/radio-group.ts

```diff
@@ -39,6 +87,63 @@ export default class SlRadioGroup extends LitElement {
     });
   }
 
+  getAllRadios(): SlRadio[] {
+    return [...this.querySelectorAll('sl-radio')];
+  }
+
+  checkRadioByIndex(index: number): SlRadio[] {
```

Comment:
Should this be documented and exposed to the users?

Reply:
Nah. They control the slot so it doesn’t seem valuable.